uber/templates/preregistration/attendee_donation_form.html

```diff
@@ -4,14 +4,15 @@
 {% block content %}
 <div class="masthead"></div>
 <div class="panel panel-default">
-{% if attendee.paid == c.NOT_PAID %}
-    <h2> Badge Payment for {{ attendee.full_name }} </h2>
+  <div class="panel-body">
```

Comment:
Here I am just fixing a layout issue that I missed in an earlier pull request. Happy to move this to its own pull request, if anyone would like to keep this pull request super specific.